pb/transaction.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package pb;
+
+message Transaction {
```

Comment:
I think instead of `Transaction`, how about using `Message`? (like more abstracted name) because we're going to send each other the **batch of transaction**

Reply:
This is an out of context question, is not it a **piece of transaction** between nodes? **batch of transaction** is between component(ACS, RBC, BBA).

And i agree with you : `Message`

Reply:
But I think rbc, bba eventually send bot (batch of transaction) to other nodes using transport